modules/vye/app/serializers/vye/claimant_verification_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Vye
+  class ClaimantVerificationSerializer < Vye::VyeSerializer
+    def serializable_hash
+      {
+        claimant_id: @resource.claimant_id,
```

Comment:
What if resource is nil? `@resource&.claimant_id`

Reply:
Fixed in this serializer and other serializers needing this fix.